src/devices/Max7219/samples/Max7219.sample.cs

```diff
@@ -37,29 +36,23 @@ static void Main(string[] args)
 
             var connectionSettings = new SpiConnectionSettings(0, 0)
             {
-                ClockFrequency = Max7219.SpiClockFrequency,
-                Mode = Max7219.SpiMode
+                ClockFrequency = Iot.Device.Max7219.Max7219.SpiClockFrequency,
+                Mode = Iot.Device.Max7219.Max7219.SpiMode
             };
             var spi = SpiDevice.Create(connectionSettings);
-            using (var devices = new Max7219(spi, cascadedDevices: 4))
+            using (var devices = new Max7219.Max7219(spi, cascadedDevices: 4))
             {
-                //initialize the devices
+                // initialize the devices
                 devices.Init();
 
                 // reinitialize the devices
                 Console.WriteLine("Init");
                 devices.Init();
 
                 // write a smiley to devices buffer
-                var smiley = new byte[] {
-                    0b00111100,
-                    0b01000010,
-                    0b10100101,
-                    0b10000001,
-                    0b10100101,
-                    0b10011001,
-                    0b01000010,
-                    0b00111100
+                var smiley = new byte[]
+                {
+                    0b00111100, 0b01000010, 0b10100101, 0b10000001, 0b10100101, 0b10011001, 0b01000010, 0b00111100
```

Comment:
what's this rule? I preferred the previous style to be honest

Reply:
I'd rather disable it here since these form and image

Reply:
This is no rule. I suppose this happened due to autoformatting the file. Not sure what exact rules applied there. I'll revert it.